packages/server/postgres/queries/src/upsertTeamPromptResponsesQuery.sql

```diff
@@ -0,0 +1,10 @@
+/*
+  @name upsertTeamPromptResponsesQuery
+  @param responses -> ((meetingId, userId, sortOrder, content, plaintextContent)...)
+*/
+INSERT INTO "TeamPromptResponse" ("meetingId", "userId", "sortOrder", "content", "plaintextContent")
+VALUES :responses
+ON CONFLICT ("meetingId", "userId") DO UPDATE SET
```

Comment:
-2 it's on me but there's no unique constraint for both `meetingId` and `userId` so this insert fails with 
```
Query 1 ERROR: ERROR:  there is no unique or exclusion constraint matching the ON CONFLICT specification
```

I wish we could simply use here the `id`. 
1. If `inputTeamPromptResponseId` is provided, provide use it
2. if `inputTeamPromptResponseId` is _not_ provided, use `DEFAULT`

I don't think it's possible with `pgtyped` but it'd be the cleanest way IMO. otherwise, we need another migration to create unique constraint between `meetingId` and `userId`

Reply:
Oh darn! It's actually my bad. I got that error before and as a temporary work around to test other things, I added the unique constraint manually to the DB and since then forgot I had that in place...

Reply:
seems no easy way to do it so I just added a migration

Reply:
FWIW I think it's worth enforcing this uniqueness constraint, even if it means adding a new migration, and even if there's a workaround using `id`.  I have tension around not enforcing uniqueness for things that we know to be unique.